doc/src/compute_efield_wolf_atom.rst

```diff
@@ -0,0 +1,126 @@
+.. index:: compute efield/wolf/atom
+
+compute efield/wolf/atom command
+================================
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   compute ID group-ID efield/wolf/atom alpha keyword val
+
+* ID, group-ID are documented in :doc:`compute <compute>` command
+* efield/atom/wolf = style name of this compute command
+* alpha = damping parameter (inverse distance units)
+* zero or more keyword/value pairs may be appended
+* keyword = *limit* or *cutoff*
+
+  .. parsed-literal::
+
+     *limit* group2-ID = limit computing the electric field contributions to a group (default: all)
+     *cutoff* value = set cutoff for computing contributions to this value (default: maximum cutoff of pair style)
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   compute 1 all efield/wolf/atom 0.2
+   compute 1 mols efield/wolf/atom 0.25 limit water cutoff 10.0
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+Define a computation that approximates the electric field at each atom in a group.
+
+.. math::
+
+   \vec{E}_i = \frac{\vec{F}coul_i}{q_i} = \sum_{j \neq i} \frac{q_j}{r_{ij}^2} \qquad r < r_c
+
+The electric field at the position of the atom *i* is the coulomb force
+on a unit charge at that point, which is equivalent to dividing the
+Coulomb force by the charge of the individual atom.
+
+In this compute the electric field is approximated as the derivative of
+the potential energy using the Wolf summation method, described in
+:ref:`Wolf <Wolf4>`, given by:
+
+.. math::
+   E_i = \frac{1}{2} \sum_{j \neq i}
+   \frac{q_i q_j {\rm erfc}(\alpha r_{ij})}{r_{ij}} +
+   \frac{1}{2} \sum_{j \neq i}
+   \frac{q_i q_j {\rm erf}(\alpha r_{ij})}{r_{ij}} \qquad r < r_c
+
+where :math:`\alpha` is the damping parameter, and *erf()* and *erfc()*
+are error-function and complementary error-function terms.  This
+potential is essentially a short-range, spherically-truncated,
+charge-neutralized, force-shifted, pairwise *1/r* summation.  With a
+manipulation of adding and subtracting a self term (for i = j) to the
+first and second term on the right-hand-side, respectively, and a small
+enough :math:`\alpha` damping parameter, the second term shrinks and the
+potential becomes a rapidly-converging real-space summation.  With a
```

Comment:
Is this statement about accuracy of Wolf correct or are there additional caveats?
I thought Wolf summation might only be appropriate for certain kinds of materials,
e.g. solids, and not liquids or soft matter (biomoleulce, biomembranes) with large asymmetries?  If so, some mention of those caveats would be useful.  Otherwise users
might think this always gives good Efield values.

Reply:
@sjplimp This was taken verbatim from the documentation of the corresponding pair style. I have been careful to put the statement "Define a computation that approximates the electric field" at the beginning of the doc page and thus already alerting people. For as long as it is not used for computing forces it is actually not so bad since errors do not accumulate and the structure will still follow the (hopefully more accurate) choice of pair/kspace style.